### ISSUE TITLE
Alles dat nog gefixed/gedaan moet worden

### ISSUE THREAD

**Issue by Xenolel** [mrTsjolder/SWOP-KULAK#42]
foutjes:
- ~~lunch break kiezen kan parse errors geven~~
- ~~?List all projects? en ?Modify system time? buttons moeten juiste error geven wanneer je niet ingelogd bent~~
- ~~laad init file -> login -> amsterdam -> man gear -> list all projects -> "project z" -> taak met id 6 -> lege error~~
- ~~PLantaskCommand reserveert te veel resources~~
- ~~Delegate task use case toont ook de branch office waartoe de geselecteerde taak al assigned is (nie echt belangrijk mja :p)~~
- ~~Alle niet finished en failed taken oplijsten in updateTaskstatus~~
- Kloppen alle overgangen van stati?
- ~~Planning.update moet zorgen dat Executing -> Available/Unavailable gaat indien de taak niet langer een planning heeft~~
- ~~? De timespan die gegeven wordt aan Task.Finish heeft geen relatie met de timespan van de planning of het moment waarop naar Executing is gegaan.~~
- ~~(niet belangrijk) je kan 10x op login knop drukken om 10 login vensters te openen~~
- ~~Het is slechts mogelijk om 1 developer te requiren door standard requirements~~
- ~~Heeft developer de juiste beschikbaarheid? wss niet~~
- ~~Met de file uit dropbox, krijg ik een error bij initialisatie: "requirements not met"~~
  - ~~Om een onverklaarbare reden wordt een extra developer toegevoegd aan de requirements.~~
- ~~Unplanned execution bestaat nog niet~~
- Unplanned execution zal conflictexceptions geven

todo:
- sequence diagrams
- class diagrams
- init files
  - scenario om resolve conflict te tonen
  - scenario om het vrijgeven van een planning te tonen bij het aanpassen van de systeemtijd
  - ... 
- tests
  - ResourceType is nog niet genoeg gecovered
- ~~scenario test delegate task~~
- ~~scenariotest updateTask~~
- modifiers

**Comment by mrTsjolder** [comment]
Alle TODO's in de code zouden nog eens overlopen moeten worden!!! 
Er zijn veel triviale dingen, maar die kunnen misschien nog op problemen wijzen...

**Comment by isostorm** [comment]
Ja, daarvoor dient diende die Skype sessie. Ik ben altijd beschikbaar, ge moe ma laten weten wnr.
